app/policies/transaction_policy.rb

```diff
@@ -0,0 +1,35 @@
+class TransactionPolicy < ApplicationPolicy
+  def create?
+    if { draft: false }
+      user.admin?
+    end
+  end
+
+  def edit?
+    if { draft: false }
+      user.admin?
+    end
+  end
+
+  def destroy?
+    true
```

Comment:
For now all users can delete as long as draft.

Reply:
Noted